jerry-core/ecma/builtin-objects/ecma-builtin-string.c

```diff
@@ -387,6 +403,42 @@ ecma_builtin_string_dispatch_construct (const ecma_value_t *arguments_list_p, /*
   return ecma_op_create_string_object (arguments_list_p, arguments_list_len);
 } /* ecma_builtin_string_dispatch_construct */
 
+/**
+ * Dispatcher of the built-in's routines
+ *
+ * @return ecma value
+ *         Returned value must be freed with ecma_free_value.
+ */
+ecma_value_t
+ecma_builtin_string_dispatch_routine (uint8_t builtin_routine_id, /**< built-in wide routine identifier */
+                                      ecma_value_t this_arg, /**< 'this' argument value */
+                                      const ecma_value_t arguments_list_p[], /**< list of arguments
+                                                                              *   passed to routine */
+                                      uint32_t arguments_number) /**< length of arguments' list */
+{
+  switch (builtin_routine_id)
+  {
+    case  ECMA_BUILTIN_STRING_OBJECT_FROM_CHAR_CODE:
+    {
+      return ecma_builtin_string_object_from_char_code (this_arg, arguments_list_p, arguments_number);
+    }
+#if JERRY_ESNEXT
+    case ECMA_BUILTIN_STRING_OBJECT_FROM_CODE_POINT:
+    {
+      return ecma_builtin_string_object_from_code_point (this_arg, arguments_list_p, arguments_number);
+    }
+    case ECMA_BUILTIN_STRING_OBJECT_RAW:
+    {
+      return ecma_builtin_string_object_raw (this_arg, arguments_list_p, arguments_number);
```

Comment:
argument this is all unused in all these three functions, and the behavior is same when no arguments is passed. This common part should be the part of the dispatcher